src/iso19111/metadata.cpp

```diff
@@ -238,6 +238,25 @@ GeographicBoundingBoxNNPtr GeographicBoundingBox::create(double west,
         throw InvalidValueTypeException(
             "GeographicBoundingBox::create() does not accept NaN values");
     }
+    if (south > north) {
+        throw InvalidValueTypeException(
+            "GeographicBoundingBox::create() does not accept south > north");
+    }
+    if (west == east && south == north) {
```

Comment:
diving in two statements 
if (west == east ) ...
if (south == north) ...
it would also support degenerated lines

Reply:
done